doc/glossary.rst

```diff
@@ -137,6 +137,14 @@ general neuroimaging concepts. If you think a term is missing, please consider
     label
         A :class:`Label` refers to a region in the cortex, also often called
         a region of interest (ROI) in the literature.
+        
+    minimum-norm estimation
+        Minimum-norm estimation (abbr. ''MNE'') can be used to generate a distributed 
+        map of activation on a :term:'source space', usually on a cortical surface. 
```

Comment:
probably need backticks rather than single quotes on these to get it to render properly in Sphinx